docs/endpoints.rst

```diff
@@ -146,6 +146,51 @@ Endpoints can be the following states:
   necessary endpoint cleanup, since it was not stopped correctly previously.
 
 
+Ensuring execution environment
+------------------------------
+
+When running a function, endpoint worker processes expect to have all the necessary
+dependencies readily available to them. For example, if a function uses ``numpy`` to do
+some calculations, and a worker is running on a machine without ``numpy`` installed, any
+attempts to execute that function using that worker will result in an error.
+
+In HPC contexts, the endpoint process - which receives tasks from the Compute central
+services and queues them up for execution - generally runs on a separate node from the
```

Comment:
Editorial bikeshed: the hyphen there is meant to be an em-dash, yeah?  Does reST support either of the following?

- `--`  (literally, _two_ dashes)
- `&mdash;`  (HTML "em-dash" entity)

The latter, if supported by reST, is bog-standard, so will render as expected ~everywhere.  I don't know that HTML is smart enough to insert non-breaking spaces on either side, so if you were feeling extra bikesheddy, then `... process&nbsp;&mdash;&nbsp;which ...` would prevent the dash from being orphaned at the beginning or end of a line.